CMakeLists.txt

```diff
@@ -1,36 +1,59 @@
 # Copyright 2010, Olivier Stasse, JRL, CNRS/AIST
-
 CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
 
-INCLUDE(cmake/base.cmake)
-INCLUDE(cmake/boost.cmake)
-INCLUDE(cmake/eigen.cmake)
-INCLUDE(cmake/pthread.cmake)
-
 SET(PROJECT_ORG stack-of-tasks)
 SET(PROJECT_NAME dynamic-graph)
 SET(PROJECT_DESCRIPTION "Dynamic graph library")
 SET(PROJECT_URL "http://github.com/${PROJECT_ORG}/${PROJECT_NAME}")
 SET(PROJECT_SUFFIX "-v3")
 
-SET(CUSTOM_HEADER_DIR "${PROJECT_NAME}")
+# Export CMake Target
+SET(PROJECT_USE_CMAKE_EXPORT TRUE)
+
+# Make sure that every header is generated in dynamic-graph
+SET(CUSTOM_HEADER_DIR ${PROJECT_NAME})
+
+# Do not fail on warning
+SET(CXX_DISABLE_WERROR TRUE)
 
+INCLUDE(cmake/base.cmake)
+
+# Specify the project.
+cmake_policy(SET CMP0048 NEW)
```

Comment:
The current version of the jrl-cmakemodules allows us to:
```cmake
COMPUTE_PROJECT_ARGS(PROJECT_ARGS LANGUAGES CXX)
PROJECT(${PROJECT_NAME} ${PROJECT_ARGS})
```

To correctly computes the project's language, version and description, for all versions of CMake.

Reply:
Done